website/src/components/ComponentsGrid.astro

```diff
@@ -2,6 +2,6 @@
 
 ---
 
-<div class='grid grid-cols-[repeat(auto-fit,minmax(min(calc(100vw-16px),650px),1fr))] gap-x-4 gap-y-6'>
+<div class='grid grid-cols-[repeat(auto-fit,minmax(min(calc(100vw-16px),650px),1fr))] content-start gap-x-4 gap-y-6'>
```

Comment:
I changed this, because the filter now increases the page size. The grid then tries to fill the elements evenly on the page. For only 3 graphs, this then no longer looks nice.